app/controllers/user_confirmations_controller.rb

```diff
@@ -0,0 +1,37 @@
+class UserConfirmationsController < DeviseController
+  include Spree::Core::ControllerHelpers::Auth # Needed for access to current_ability, so we can authorize! actions
+
+  # GET /resource/confirmation/new
```

Comment:
Yeah the expliciter de better 😂 . I would also like to know what kind of customization we need on top of the already customized Spree/Devise